aip/cloud/new.md

```diff
@@ -0,0 +1,123 @@
+---
+aip:
+  id: 2511
+  scope: cloud
+  state: draft
+  created: 2019-11-22
+permlink: /cloud/2511
+redirect_from:
+  - /2511
+---
+
+<!--
+TODO: discuss policy lifecycle? binding lifecycle?
+TODO: discuss policy conditions.
+-->
+
+# Policy API Guidance
+
+A policy is a document-based resource that, when set on a resource, controls
```

Comment:
When you use the term "resource" here (the first time), are you using it consistent with the [AIP-121](http://aip.dev/121) definition of the word? (I _think_ the answer is no, but I am uncertain.)

(I am fairly confident that the remaining uses of the term are all the AIP-121 definition.)